packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -374,6 +375,10 @@ class Doctor {
     }
     bool doctorResult = true;
     int issues = 0;
+    
+    // This timestamp will be used on the backend of GA4 to group each of the events that
+    // were sent for each doctor validator and its result
+    final int analyticsTimestamp = globals.systemClock.now().millisecondsSinceEpoch;
```

Comment:
please inject systemClock, we should not be adding more globals

Reply:
Done! 👍